engine/src/main/java/org/terasology/rendering/dag/nodes/ShadowMapNode.java

```diff
@@ -81,6 +80,8 @@ public void initialise() {
 
         addDesiredStateChange(new BindFBO(SCENE_SHADOW_MAP));
         addDesiredStateChange(new SetViewportSizeOf(SCENE_SHADOW_MAP));
+        addDesiredStateChange(new Clear(true, true, false));
+        addDesiredStateChange(new DisableCapability(GL_CULL_FACE));
```

Comment:
In light of what I wrote yesterday I think this should specifically become something like 

`new SetCullFace(enabled)`